components/kubearchive/base/migration-job.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: kubearchive-schema-migration-
+  namespace: kubearchive
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+spec:
+  parallelism: 1
+  backoffLimit: 4
+  template:
+    spec:
+      containers:
+        - name: migration
+          image: quay.io/fedora/python-311:20240911
+          command: ["/bin/sh", "-c"]
+          args:
+            - git clone https://github.com/kubearchive/kubearchive --depth=1 --branch=${KUBEARCHIVE_VERSION} /tmp/kubearchive;
+              cd /tmp/kubearchive;
+              curl -L https://github.com/golang-migrate/migrate/releases/latest/download/migrate.linux-amd64.tar.gz | tar xvz migrate;
```

Comment:
should we stick to a fixed version instead of `latest`?

Reply:
Done! Used the latest release available.